Model/Resizer.php

```diff
@@ -114,27 +114,38 @@ public function __construct(
     }
 
     /**
-     * Resized image and return url
-     * - Return original image url if no success
+     * Resize image
      *
      * @param string $imageUrl
      * @param null|int $width
      * @param null|int $height
      * @param array $resizeSettings
-     * @return bool|string
+     * @return self
      */
-    public function resizeAndGetUrl(string $imageUrl, $width, $height, array $resizeSettings = [])
+    public function resize(string $imageUrl, $width, $height, array $resizeSettings = [])
```

Comment:
I need to refactor this method. I should get rid of all these `init` methods that create temporal coupling, which should be avoided. I would try to improve that after merging your PR.

Reply:
It'd be nice if it would return value objects or something, right?